recipes-omnect/initrdscripts/omnect-os-initramfs/grub-sh

```diff
@@ -19,19 +19,49 @@ get_bootloader_env_var()
     test_and_mount_boot || return 1
     local value=$(grub-editenv ${ROOTFS_DIR}/boot/EFI/BOOT/grubenv list | grep ${var_name}\= | cut -d'=' -f2)
```

Comment:
I know that didn't change, but anyway: is this prefix secure?
I would have expected something like "grep '\<'${var_name}=" or, in case the listing is one variable per line, "grep '^'${var_name}="

Reply:
adapted ... still not "save", e.g. we may not use multiline values